recipes/spacy-pkuseg/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "spacy-pkuseg" %}
+{% set modulename = "spacy_pkuseg" %}
+{% set version = "0.0.27" %}
+{% set sha256sum = "0ab597e0eea9357ffa21e9ccba0f06af07db6b48cbe3cbb811916b68f5284a38" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/s/{{ name }}/{{ modulename }}-{{ version }}.tar.gz
+  sha256: {{ sha256sum }}
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  skip: true  # [py2k or py35]
+
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cython
+    - numpy >=1.16.0
+  host:
+    - python
+    - pip
+    - setuptools
+    - wheel
+    - cython
+    - numpy >=1.16.0
```

Comment:
If you are linking against numpy: https://conda-forge.org/docs/maintainer/knowledge_base.html#building-against-numpy

Reply:
Thanks for the feedback, I hope the update fixed this correctly?